setup.py

```diff
@@ -17,7 +17,7 @@
         ' the same time and control them from a single command prompt.'),
     entry_points={
         'console_scripts': [
-            'polysh=polysh.main:main',
+            'polysh=polysh.main:raven_main',
```

Comment:
This sounds a little strange.  Maybe we should keep it as `main()` and call the next one something like `run()`.

Reply:
Then we would have main() calling run() calling main_loop(), which is also strange.